community/openebs/templates/deployment-maya-provisioner.yaml

```diff
@@ -81,6 +81,7 @@ spec:
       tolerations:
 {{ toYaml .Values.provisioner.tolerations | indent 8 }}
 {{- end }}
+{{- if .Values.provisioner.affinity }}

```

Comment:
@ChandanSagar this change is not required here, we can remove it.

Reply:
Done.